rstcheck.py

```diff
@@ -44,6 +44,12 @@
 import docutils.utils
 import docutils.writers
 
+try:
+    from sphinx.roles import *  # NOQA
```

Comment:
I added a few examples in tests, see https://github.com/myint/rstcheck/commit/47e9727c4bcf035ef7ed6f4ee6c1ee0a75b58702

Ignoring directives need you for that would will not to set sphinx environment. Some roles are extensions, so it is also better to ignore locally.